src/processmoduleconfig/update_s390x.go

```diff
@@ -0,0 +1,141 @@
+//go:build s390x
+//+build s390x
+
+package processmoduleconfig
```

Comment:
wouldn't it be enough to have 
`func createProcessModuleConfigFile(fs afero.Fs, destPath string) error {` in a separate files arch dependent files, so we minimize duplicate logic ?

Reply:
@0sewa0 I pushed new commit to correct this. I couldn't get around duplicating `createProcessModuleConfigFile()` for both s390x and other architectures, but the rest of the duplicate code was removed.